contestants.json

```diff
@@ -256,5 +256,13 @@
 	   "github" : "https://github.com/utkarshsingh99",
 	   "twitter" : ""
       }  
+      {
+	     "name":"Komal Bhadani",
+		 "imageurl":"contestant/Pic.jpg",
```

Comment:
You are referring to a image here. But there is no such pic. want to upload one?